core/templates/components/oppia-angular-root.component.ts

```diff
@@ -437,6 +437,8 @@ import { SubtopicPageObjectFactory } from
   'domain/topic/SubtopicPageObjectFactory';
 import { SubtopicViewerBackendApiService } from
   'domain/subtopic_viewer/subtopic-viewer-backend-api.service';
+import { SuggestionModalForCreatorDashboardBackendApiService } from
```

Comment:
You no longer need to add migrated services here. Please remove it.

Reply:
Done.